common/models/task.go

```diff
@@ -1,15 +1,39 @@
 package models
 
+import swagger "github.com/zpatrick/go-plugin-swagger"
+
 type Task struct {
-	Copies          []TaskCopy `json:"copies"`
-	DeployID        string     `json:"deploy_id"`
-	DeployName      string     `json:"deploy_name"`
-	DeployVersion   string     `json:"deploy_version"`
-	DesiredCount    int64      `json:"desired_count"`
-	EnvironmentID   string     `json:"environment_id"`
-	EnvironmentName string     `json:"environment_name"`
-	PendingCount    int64      `json:"pending_count"`
-	RunningCount    int64      `json:"running_count"`
-	TaskID          string     `json:"task_id"`
-	TaskName        string     `json:"task_name"`
+	TaskID          string `json:"task_id"`
+	TaskName        string `json:"task_name"`
+	EnvironmentID   string `json:"environment_id"`
+	EnvironmentName string `json:"environment_name"`
+	DeployID        string `json:"deploy_id"`
+	DeployName      string `json:"deploy_name"`
+	DeployVersion   int    `json:"deploy_version"`
+
+	// todo: fix copies field
+	//Copies       []TaskCopy `json:"copies"`
```

Comment:
We should re-visit this at some point. This is a very leaky abstraction in the model. I think it may be better to change this to something much more layer0-centric, e.g. a slice of objects that have the container name and exit code?

```
type Task struct {
    ...
    Things []TaskThing
}

type TaskThing struct {
    ContainerName string
    ExitCode int
    // meta?
}
```

Reply:
Transforming what's currently just the output from ECS into a model more in-line with the rest of our implementation sounds like a fine idea to me. :+1:

Reply:
Is there an object we can borrow from the AWS SDK? Otherwise, would it be overkill to add this model to our models package?

Reply:
My point is to move away from that. We basically do that currently, and it's a leaky abstraction that doesn't map well to how layer0 does tasks.